.github/workflows/create-reports.yml

```diff
@@ -15,8 +19,8 @@ jobs:
       tools_table: ${{ steps.tools_table.outputs.tools-table }}
       pk_sim_version: ${{ steps.tools_table.outputs.pk-sim }}
       qualification_framework_version: ${{ steps.tools_table.outputs.qualification-framework }}
-      models_table: ${{ steps.models_table.outputs.table }}
-      models: ${{steps.models.outputs.data}}
+      qualifications_table: ${{ steps.qualifications_table.outputs.table }}
+      qualifications: ${{steps.qualifications.outputs.data}}
```

Comment:
renamed models.csv to qualifications.csv and also renamed model to qualification in code